### ISSUE TITLE
Chromium version (57) is beginning to show its age

### ISSUE THREAD

**Issue by cdata** [google-ar/WebARonARCore#18]
The latest stable Chrome release as of this writing is 61. There have been a number of notable features landed in the versions since 57. It would be nice to be able to write experiments that pair other emerging browser features with WebAR.

Some features that have been released in stable since 57 include IndexedDB 2.0, Web Share API and support for JavaScript modules.

**Comment by larsgk** [comment]
+1

Also, I'd really like to build some WebBluetooth and WebUSB demos for ChromeDev Summit utilizing it - if possible?

**Comment by judax** [comment]
Sorry for the late reply. We are working on trying to get up to date with a more current Chromium version but it is harder that expected. Will try to update this issue when we do have a new version. Thanks for providing your feedback.

**Comment by cdata** [comment]
@judax was this closed because there is now a way forward for those of us who want to use up to date platform features?

**Comment by jsantell** [comment]
@cdata a modern Chromium build (non-webview, tracking the latest Chromium changes) is in the pipes (no ETA yet)

**Comment by cdata** [comment]
Then it seems like this issue should remain open.

**Comment by judax** [comment]
Agreed. Reopening.

**Comment by robertjuh** [comment]
Amazing, good luck with it!

**Comment by anutin544** [comment]
Ok

**Comment by robertjuh** [comment]
Will this mean it will work on normal chrome browsers in the future as well? 
Or will this never happen because of let's say security matters?

I'm trying to find the answer to this for making a decision for my bachelor project, i saw @judax mention that it will come to the latest chromium eventually (?) which would already be a great progress!

**Comment by judax** [comment]
We are working on updating this projects completely to provide a more up to date ARCore and Chromium support.

**Comment by robertjuh** [comment]
Yes i know, is there any roadmap or anything available? It would be useful to know if we will have let's say chrome, safari and firefox in 2019 for example or if that is not a technical possibility because of security / compliance problems.

Either way, i love this project! keep it up
